files-community/Files
# Feature: Added open in VS/VS Code to status bar

**Resolved / Related Issues**
   Closes #12632

**Validation**
How did you test these changes?
   1. Open app and navigate to a directory containing a `.sln` file
   2. Click on `Open in VS`
   3. Navigate to a git directory
   4. Click on `Open in VS Code`
